7zip/tests/test.ps1

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env pwsh
```

Comment:
Is this correct for Powershell?    Does Powershell on windows ignore this line?

Reply:
It does. Only linux will honor the shebang